src/svd/__init__.py

```diff
@@ -94,7 +113,12 @@
     "RegisterType",
     "Struct",
     # other
+    "log",
+    "util",
     "__version__",
 ]
 
-del importlib.metadata
+if not typing.TYPE_CHECKING:
+    del importlib.metadata
```

Comment:
This deletes `metadata` from `importlib`, which ends up breaking other packages (including nrf-regtool).
If the intent is to "un-import" stuff from this scope, then this doesn't seem to be the right way to do it.
```suggestion
```